let p = Promise.resolve(new Error("NÃ£o deu certo"));

console.log("wasd");

p.then(value => console.log(value))
    .catch(reason => console.log("Falhou "+reason))